components/images.css

```diff
@@ -0,0 +1,24 @@
+.img-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+.img-responsive-container{
+    display: flex;
```

Comment:
Try to combine classes in to one when you have same property to use. Like here display: flex